sdcm/utils/version_utils.py

```diff
@@ -414,6 +416,43 @@ def resolve_latest_repo_symlink(url: str) -> str:
     return resolved_url
 
 
+def get_specific_tag_of_docker_image(docker_repo: str):
+
+    def get_disget(tag_dict):
+        for image in tag_dict['images']:
+            if image['architecture'] == arch:
+                return image['digest']
+        return None
+
+    def get_last_update_date(tag_dict):
+        return datetime.datetime.fromisoformat(tag_dict['last_updated'].split('T')[0])
+
+    arch = 'amd64'
+    url = 'https://hub.docker.com/v2/repositories/{}/tags/{}'
+    docker_latest = requests.get(url.format(docker_repo, 'latest')).json()
+    latest_number_of_images = len(docker_latest['images'])
+    latest_update_time = get_last_update_date(docker_latest)
+    latest_digest = get_disget(docker_latest)
+    for page_number in count(start=1):
+        all_tags = requests.get(url=f'https://hub.docker.com/v2/repositories/{docker_repo}/'
+                                    f'tags?page_size=25&page={page_number}').json()
+        if 'results' not in all_tags:
+            break
+        current_update_time = get_last_update_date(all_tags['results'][0])
```

Comment:
Are you sure that there is always at least one item exists?
I am not and in such a case it becomes `not safe` programming.
So, please, add a check for the element presence.

Reply:
moved this into it's own PR #4971